### ISSUE TITLE
View is not accessible - Not hovering / announcing month names when talkback is on

### ISSUE THREAD

**Issue by hishamMuneer** [premkumarroyal/MonthAndYearPicker#19]
View is not accessible - Not hovering / announcing month names when talkback is on. 

A blind user is not able to select correct month when picker is open. 
Steps: 
1. Open talkback on your phone by pressing and holding both volume buttons.
2. Open picker view and swipe right to move to next item.
3. In monthView not able to focus/select month
